README.md

```diff
@@ -171,6 +190,24 @@ sudo apt update
 sudo apt install rdiff-backup
 ```
 
+### Fedora, CentOS and RHEL
```

Comment:
Are you sure the "level" are the irght one here ? I would have put "####"

Reply:
Well spotted and fixed.